rules.ini

```diff
@@ -186,6 +186,7 @@ OpenXR = (?:^|/)(?:lib)?openxr_loader\.(?:dll|so)$
 Photon = (?:^|/)Photon(?:3Unity3D|3DotNet)\.dll$
 Qt = (?:^|/)Qt(?:Core4|[0-9]Core)\.dll$
 SDL = (?:^|/)sdl2?\.dll$
+Sodium = (?:^|/)libsodium\.(?:dll|so|dylib)$
```

Comment:
you can just check for .dll ; .so & .dylib don't catch anything that doesn't get caught by .dll

Reply:
i just used all files that were listed in issue, but i can change it